apps/component-date/fields/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const dateComponent = require('hof-component-date');
+
+module.exports = {
+  'date-of-birth': dateComponent('date-of-birth', {
+    // before means a date cannot be in the future
+  }),
+  'passport-expiry': dateComponent('passport-expiry', {
+    // after means a date cannot be in the past
+    validate: ['required', 'after']
```

Comment:
It'd be good to include an example of `before`/`after` validation with an offset parameter. Mostly because I can't even remember myself how that works.